airflow/ti_deps/deps/pool_slots_available_dep.py

```diff
@@ -20,9 +20,7 @@
 
 from airflow.ti_deps.deps.base_ti_dep import BaseTIDep
 from airflow.utils.session import provide_session
-from airflow.utils.state import State
-
-STATES_TO_COUNT_AS_RUNNING = [State.RUNNING, State.QUEUED]
```

Comment:
I like this change but it could be better to make this as separate PR, WDYT @mik-laj ?

Reply:
In my opinion, it could be in this PR.